### ISSUE TITLE
Appropedia list of articles seem to include deleted articles

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#1534]
'GSTC1 test page' is a deleted page, but it seems to be still listed in the namespace and then MWoffliner tries somehow to retrieve it. See https://farm.openzim.org/pipeline/020ccb7605803f00747fc216/debug.

Nothing related to this article should be scraped.

**Comment by kelson42** [comment]
Now the problem is different, article `Las_Malvinas_ecoladrillo_schoolroom_2012/Timeline` seems to not render quick enough.

**Comment by Sophivorus** [comment]
@kelson42 Hi, thanks for the follow up! I just edited https://www.appropedia.org/Las_Malvinas_ecoladrillo_schoolroom_2012/Timeline to reduce the number of template calls and other complexities. I also went through the various maintenance reports and categories on Appropedia to fix other issues that could cause trouble. Can we try again? Also, is there some way to get a list of pages that need fixing, or maybe configuring the script to skip the problematic pages rather than halt completely? Thanks again, cheers!!
